src/Compiler/Checking/PostInferenceChecks.fs

```diff
@@ -2101,6 +2101,7 @@ let CheckModuleBinding cenv env (TBind(v, e, _) as bind) =
     let isExplicitEntryPoint = HasFSharpAttribute g g.attrib_EntryPointAttribute v.Attribs
     if isExplicitEntryPoint then 
         cenv.entryPointGiven <- true
+        warning (Error(FSIstrings.SR.fsiEntryPointWontBeInvokedAutomatically(v.DisplayName, v.DisplayName, v.DisplayName), v.Range))
```

Comment:
This will affect non-scripts too, won't it?

Reply:
I could not run the test locally for some reason :( So I used the CI to see how this affects the rest of the test .

Reply:
@vzarytovskii  I’m not sure where to raise the warning only for the scripts TBH . So if someone is already on it happy to close this PR

Reply:
Sure. go for it :) . Will learn from your PR